classes/Authentication/SAML/XDSamlAuthentication.php

```diff
@@ -84,10 +93,20 @@ public function __construct()
      */
     public function isSamlConfigured()
     {
-        $this->_isConfigured = count($this->_sources) > 0 ? true : false;
+        if($this->_isConfigured === null){
+            $this->_isConfigured = (count($this->_sources));
```

Comment:
Couldn't this just be:

```suggestion
            return count($this->_sources);
```

And then we can get rid of the variable.

Reply:
answered in other comment